Gemfile

```diff
@@ -9,8 +9,18 @@ gem 'sass-rails', '~> 5.0'
 gem 'uglifier', '>= 1.3.0'
 gem 'turbolinks', '~> 5'
 gem 'bootsnap', '>= 1.1.0', require: false
+gem 'haml-rails', '~> 1.0'
```

Comment:
A bit out of scope. This PR is about setting up test frameworks. Not a blocker, but I just wanted to point this out.